nengo/dists.py

```diff
@@ -189,20 +189,29 @@ class UniformHypersphere(Distribution):
 
     Parameters
     ----------
+    min_magnitude : Number
+        The closed lower bound of the uniform hypersphere distribution;
+        such that the sampled vector magnitudes >= min_magnitude.
+        The default value and minimum value for min_magnitude is 0.
+        The maximum value for min_magnitude is 1. Values outside this range
+        will be clipped to this range.
```

Comment:
Can this be more concise? e.g. "Lower bound on the returned vector magnitudes (i.e. magnitude >= min_magnitude). Must be in the range [0, 1]; defaults to 0. Ignored if `surface == True`."